### ISSUE TITLE
don't allow IPI evidence with catalytic MF

### ISSUE THREAD

**Issue by Antonialock** [pombase/canto#1503]
can pairing IPI evidence with a catalytic MF be blocked (it is fine for a 'binding' MF)?

**Comment by kimrutherford** [comment]
> can pairing IPI evidence with a catalytic MF be blocked (it is fine for a 'binding' MF)?

Yep, we can make that configurable.  Are the any other evidence-term pairings that should be disallowed?

**Comment by ValWood** [comment]
I think there are... Midori, are there any in the relevant GO check file?

**Comment by ValWood** [comment]
OK, I think we can eventually bring other GO checks forward. But there are probably already tickets for these if we have thought of them...
